test/configuration_mixin_test.rb

```diff
@@ -18,10 +18,8 @@
     end
 
     it "not generate keys longer that 255" do
-      very_large_number = (1..999).to_a.join
-      query_attributes  = [ [:blog_id, very_large_number] ]
-
-      assert(Post.kasket_key_for(query_attributes).size < 255)
+      Post.stubs(:quoted_value_for_column).returns((1..999).to_a.join.to_s)
+      assert(Post.kasket_key_for([:blog_id, 1]).size < 255)
```

Comment:
I'll need to revert the 4.0.0 commit and tag. This should be included first.

On Thursday, August 20, 2015, Mick Staugaard <notifications@github.com>
wrote:

> In test/configuration_mixin_test.rb
> <https://github.com/zendesk/kasket/pull/17#discussion_r37551013>:
>
> > @@ -18,10 +18,8 @@
> >      end
> >
> >      it "not generate keys longer that 255" do
> > -      very_large_number = (1..999).to_a.join
> > -      query_attributes  = [ [:blog_id, very_large_number] ]
> > -
> > -      assert(Post.kasket_key_for(query_attributes).size < 255)
> > +      Post.stubs(:quoted_value_for_column).returns((1..999).to_a.join.to_s)
> > +      assert(Post.kasket_key_for([:blog_id, 1]).size < 255)
>
> memcached will not auto-shorten keys. Some memcached clients do, but not
> all.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/kasket/pull/17/files#r37551013>.
>